sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveWindowFunctionQuerySuite.scala

```diff
@@ -749,7 +749,7 @@ abstract class HiveWindowFunctionQueryBaseSuite extends HiveComparisonTest with
       |range between current row and unbounded following) as s1
       |from part
     """.stripMargin, reset = false)
-
+  */
```

Comment:
I think this is due to the reverse order processing of these frames. The differences are truely tiny: a few ULPs.